src/Sulu/Bundle/AdminBundle/Resources/js/components/ItemSelection/ItemSelection.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+import React from 'react';
+import type {ChildrenArray, Element} from 'react';
+import {SortableContainer, SortableElement, SortableHandle, arrayMove} from 'react-sortable-hoc';
+import type {Button} from './types';
+import Header from './Header';
+import Item from './Item';
+import itemSelectionStyles from './itemSelection.scss';
+
+type Props = {
+    children: ChildrenArray<*>,
+    /** The text inside the header bar of the `ItemSelection` */
+    label?: string,
+    /** Called when the remove button is clicked on an item */
+    onItemRemove?: (itemid: string | number) => void,
+    /** Called after a drag and drop action was executed */
+    onItemsSorted?: (oldIndex: number, newIndex: number) => void,
+    /** The config of the button placed left side of the header */
+    leftButton?: Button,
+    /** The config of the button placed right side of the header */
+    rightButton?: Button,
+};
+
+export default class ItemSelection extends React.PureComponent<Props> {
+    static Item = Item;
+
+    static arrayMove = arrayMove;
+
+    createItem(originalItem: Element<typeof Item>) {
+        return (
+            <li className={itemSelectionStyles.listElement}>
+                {
+                    React.cloneElement(
+                        originalItem,
+                        {
+                            ...originalItem.props,
+                            onRemove: this.handleItemRemove,
+                            createDragHandle: this.createDragHandle,
```

Comment:
Is there a reason this is passed like this? Can't we use the `SortableHandle` directly in the `Item` component?